Tensile/Contractions.py

```diff
@@ -327,6 +327,8 @@ def CompoundPredicates(cls, state, problemType):
 
         if 'PersistentKernel' in state and state['PersistentKernel']:
             rv += [cls("PersistentKernelCheck", value = True)]
+            # debugging: set value = false to allow the problem runnable with PK
```

Comment:
See comment on the predicate:
- If wanting to debug here, just don't attach the predicate.
We can probably remove the value.

Reply:
Yes, I totally agree with you. That's also what I did before. (Not to add predict at all) I remember there was some reason why I changed it, but I can not recall...I'll change back to the better one :)